### DISCUSSION TITLE
如何创建【快捷方式】或【图标】到【桌面】或【应用抽屉】？

### DISCUSSION THREAD

**Discussion by 1457384613gh** [LSPosed/LSPosed#2149] [Q&A]
原生安卓12，
安装了模块，重启，提示是否创建快捷方式，不小心点错了，点成了不创建。
Magisk的模块列表有LSPosed，桌面没快捷方式，启动器没图标。
文件管理器找到了 /data/adb/…/manager.apk，但是不知道该怎么弄。

**Comment by mywalkb** [comment]
From README

In any case, you can dial *#*#5776733#*#* (aka LSPosed) to launch the manager if you have a dialer
You can make shortcut from settings.

or

If the shortcut cannot be added, you can install the manager located at /data/adb/lspd/manager.apk manually

I think the first sokution is good for you
